tests/playwright/src/specs/container-smoke.spec.ts

```diff
@@ -154,7 +154,61 @@ describe('Verification of container creation workflow', async () => {
     await playExpect(startButton).toBeVisible();
   });
 
-  test('Deleting a container', async () => {
+  test(`Start a container from the Containers list`, async () => {
+    const navigationBar = new NavigationBar(page);
+    const containers = await navigationBar.openContainers();
+    const containersDetails = await containers.openContainersDetails(containerToRun);
+    await playExpect(containersDetails.heading).toBeVisible();
+    await playExpect(containersDetails.heading).toContainText(containerToRun);
+    // test state of container in summary tab
+    await pdRunner.screenshot('containers-container-details-start-container.png');
+    const containerState = await containersDetails.getState();
+    playExpect(containerState).toContain(ContainerState.Exited.toLowerCase());
+
+    await navigationBar.openContainers();
+    const containerRow = await containers.getContainerRowByName(containerToRun);
+    if (containerRow === undefined) {
+      throw Error(`Container: '${containerToRun}' does not exist`);
+    }
+    const containerRowStartButton = containerRow.getByRole('button', { name: 'Start Container' });
+    await playExpect(containerRowStartButton).toBeVisible();
+    await containerRowStartButton.click();
+
+    await containers.openContainersDetails(containerToRun);
+    await playExpect
+      .poll(async () => containersDetails.getState(), { timeout: 20000 })
+      .toContain(ContainerState.Running.toLowerCase());
+    await playExpect(await containersDetails.getStateLocator()).toHaveText(ContainerState.Running.toLowerCase());
+  });
+
+  test(`Stop a container from the Containers list`, async () => {
+    const navigationBar = new NavigationBar(page);
+    const containers = await navigationBar.openContainers();
+    const containersDetails = await containers.openContainersDetails(containerToRun);
+    await playExpect(containersDetails.heading).toBeVisible();
+    await playExpect(containersDetails.heading).toContainText(containerToRun);
+    // test state of container in summary tab
+    await pdRunner.screenshot('containers-container-details-stop-container.png');
+    const containerState = await containersDetails.getState();
+    playExpect(containerState).toContain(ContainerState.Running.toLowerCase());
+
+    await navigationBar.openContainers();
+    const containerRow = await containers.getContainerRowByName(containerToRun);
```

Comment:
there is the method containerExists() in the containers-page POM for this,

Reply:
This part should be removed, right?

Reply:
yes